topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -46,16 +46,20 @@
 
 	private static final String ATTRIBUTE_APPLICATION_URL = "data-tmb-application-url";
 	private static final String ATTRIBUTE_JAVASCRIPT_PATH = "data-tmb-javascript-url";
-	private static final String DATA_TMB_COLOR = "data-tmb-color";
+	private static final String DATA_TMB_BGCOLOR = "data-tmb-bgcolor";
+	private static final String DATA_TMB_TXTCOLOR = "data-tmb-txtcolor";
 	private static final String DATA_TMB_ICON_URL = "data-tmb-icon-url";
 	private static final String DATA_TMB_HEADLINE = "data-tmb-headline";
-	private static final String DEFAULT_BACKGROUND_COLOR = "#FF0000";
+	private static final String DATA_TMB_THEME = "data-tmb-theme";
+	private static final String DEFAULT_BACKGROUND_COLOR = "#f8f8f8";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "DEFAULT_BACKGROUND_COLOR" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)